src/cplscheme/SerialCouplingScheme.hpp

```diff
@@ -50,7 +50,20 @@ class SerialCouplingScheme : public BiCouplingScheme {
       m2n::PtrM2N                   m2n,
       constants::TimesteppingMethod dtMethod,
       CouplingMode                  cplMode,
-      int                           maxIterations = UNDEFINED_MAX_ITERATIONS);
+      int                           minIterations,
+      int                           maxIterations);
```

Comment:
Why do we not use default values here anymore?

Reply:
Two defaults are super easy to misuse. So I added a second constructor for explicit cases which doesn't take iterations at all.